tests/Base.php

```diff
@@ -16,14 +17,14 @@ public function setUp(): void
 
         $pathRoot = __DIR__ . '/fixtures/java/';
 
+        ClassResolver::add([
+            [ClassResolver::RESOURCE_TYPE_FILE, __DIR__ . '/caches'],
+        ]);
+
         foreach ($this->fixtures as $fixture) {
-            exec('javac -classpath ' . $pathRoot . ':' . $pathRoot . 'caches -encoding UTF8 ' . $pathRoot . str_replace(['../', './'], '', $fixture) . '.java -d ' . __DIR__ . '/caches');
-            $this->initiatedJavaClasses[$fixture] = new \PHPJava\Core\JavaClass(
-                new JavaCompiledClass(
-                    new \PHPJava\Core\Stream\Reader\FileReader(
-                        $this->getClassName($fixture)
-                    )
-                )
+            exec('javac -classpath ' . $pathRoot . ' -encoding UTF8 ' . $pathRoot . str_replace(['../', './'], '', $fixture) . '.java -d ' . __DIR__ . '/caches');
```

Comment:
I want here to use `escapeshellarg` function. What do you think?
https://www.php.net/manual/en/function.escapeshellarg.php

Reply:
I think either way is fine.

Reply:
Go ahead then!